### PR TITLE
Addition of files used to run Woke word checker.

### PR DESCRIPTION
**Pull request by deb-intel**
Signed-off-by: Deb Taylor <deb.taylor@intel.com>

Includes two files:
.wokeignore lists files to ignore when running Woke.
rules-woke.yaml is the customized file that must be run to accurately identify and correct non-inclusive terms.
    This file is called by woke -c ./rules-woke.yaml

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@xiulipan ready when you are.

File: rules-woke.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      - main for firmware-related topics and provider for hardware-related topics
+
+  - name: blacklist
+    terms:
+      - blacklist
+      - black-list
+    alternatives:
+      - blocklist
+
+  - name: whitelist
```
Comment (xiulipan) [inline rules-woke.yaml:34]: I see duplicate `whitelist` here.

**Review by xiulipan — CHANGES_REQUESTED** [review]
I found some duplicated rules.

**Comment by xiulipan** [comment]
@deb-intel @lgirdwood I am fine with this configure files, please merge if you solve the duplicated rules above. Then I will work on this base for the woke.

**Comment by xiulipan** [comment]
@deb-intel Good news, the public market action can work with extend rules. see https://github.com/xiulipan/sof-docs/pull/1#discussion_r568308045 for detail.
I will try to modify the wokeignore to avoid such issues. But it prove that our customized rules is working.

File: .wokeignore [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# The following files can be ignored when running Woke.
+
+rules-woke.yaml
+conf.py
+*.pu
+tox.ini
+Makefile
+.travis.yml
+.github/workflows/pull-request.yml
```
Comment (xiulipan) [inline .wokeignore:9]:
```suggestion
.github/workflows/*.yml
```
I think we will add more code checked as actions later.

File: rules-woke.yaml (hunk 1 above)
Comment (deb-intel) [reply rules-woke.yaml:34]: Nice catch. Corrected

File: .wokeignore (hunk 2 above)
Comment (deb-intel) [reply .wokeignore:9]: Modified.

**Review by xiulipan — APPROVED** [review]
LGTM.
PS: do we need this for other project?

**Comment by deb-intel** [comment]
@xiulipan Are you referring to a particular project?

**Comment by lgirdwood** [comment]
> @xiulipan Are you referring to a particular project?

It's the source code for FW and tools etc, but we can do this after CNY when @xiulipan is back.

**Pull request merged**